server/partials/footer.html

```diff
@@ -22,9 +22,11 @@ <h4 class="ui header">Community</h4>
         <h4 class="ui header">Social</h4>
         <div class="ui link list">
           <a class="item" href="https://twitter.com/fomanticui" target="_blank">
-            <i class="twitter icon"></i>
-            Twitter
-          </a>
+            <svg width="1200" height="1227" viewBox="0 0 1200 1227" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path>
+            </svg>
+              <span style="vertical-align:middle">Twitter</span>
```

Comment:
No inline styling. Please enhance the css selector in docs.css instead.

```css
.footer .ui.link.list > .item svg+span {
  vertical-align: middle;
}
```

Reply:
Actually with the previous twitter icon I have to do the align 
I will make the changes

Reply:
Should remove left margin from svg tag.

![fui-docs-twitter-logo-update](https://github.com/fomantic/Fomantic-UI-Docs/assets/930315/9789500f-2bb5-4e22-a2eb-349502050df7)

Reply:
Sorry. Ignore this. The left margin makes the text align properly. I can't notice without zoom-in.

Reply:
I have done all the changes that were told to be done.